karaf/apache-brooklyn/src/main/resources/etc/default.catalog.bom

```diff
@@ -1,54 +1,366 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
 
+# this catalog bom is an illustration supplying a few useful sample items
+# and templates to get started using Brooklyn
 
 brooklyn.catalog:
-    version: 0.10.0-SNAPSHOT # BROOKLYN_VERSION
-    items:
-    - id: server-template
-      itemType: template
-      name: "Template: Server"
-      description: |
-        Sample YAML to provision a server in a cloud with illustrative VM properties
-      item:
-        name: Server (Brooklyn Example)
-
-        # this basic example shows how Brooklyn can provision a single raw VM
-        # in the cloud or location of your choice
-
-        services:
-        - type:           org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
-          name:           My VM
-
-        # location can be e.g. `softlayer` or `jclouds:openstack-nova:https://9.9.9.9:9999/v2.0/`,
-        # or `localhost` or `byon:(hosts="10.9.1.1,10.9.1.2,produser2@10.9.2.{10,11,20-29}")`
-        location:
-          jclouds:aws-ec2:
-            # edit these to use your credential (or delete if credentials specified in brooklyn.properties)
-            identity:     <REPLACE>
-            credential:   <REPLACE>
-
-            region:       eu-central-1
-
-            # we want Ubuntu, with a lot of RAM
-            osFamily:     ubuntu
-            minRam:       8gb
-
-            # set up this user and password (default is to authorize a public key)
-            user:         sample
-            password:     s4mpl3
+  version: 0.10.0-SNAPSHOT  # BROOKLYN_VERSION
+  items:
+  - brooklyn.libraries:
+    - name: org.apache.brooklyn.karaf-init
+      version: 0.10.0.SNAPSHOT # BROOKLYN_VERSION
+    include: classpath://catalog-classes.bom
 
+  - id: server
+    description: |
```

Comment:
prefer that we explicitly include `itemType: entity`. Same for `vanilla-bash-server`, `load-balancer` and `cluster`.